src/pages/Vaults/VaultsOverview/components/CreateVaults/CreateVaults.tsx

```diff
@@ -0,0 +1,57 @@
+import { useId } from '@react-aria/utils';
+import { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+
+import { H3, Modal, Stack } from '@/component-library';
+import { AvailableVaultData, useGetAvailableVaults } from '@/utils/hooks/api/use-get-available-vaults';
+import { VaultData } from '@/utils/hooks/api/vaults/get-vault-data';
+
+import { CreateVaultWizard } from '../CreateVaultWizard';
+import { VaultsTable, VaultsTableProps, VaultsTableRow } from '../VaultsTable/VaultsTable';
+
+const isVaultInstalled = (vaults: VaultData[], currentVault: AvailableVaultData) =>
```

Comment:
Not critical, but be good to have this outside of the component - we have a vault client check higher up in the app, and we should make sure we have a single utility for handling this check.

Reply:
where should this go to then?

Reply:
I think probably at the app level - we would want to consolidate this with the check for whether an account is a vault. Not a question for now though :) 
```suggestion
// Can this be consolidated with the vault account check at the app level?
const isVaultInstalled = (vaults: VaultData[], currentVault: AvailableVaultData) =>
```